src/navigation.tpl

```diff
@@ -13,11 +13,11 @@
 <!-- SECTION -->
     <li class="section"><h4 class="header toggle-target">Install & Configure</h4>
     	<ul>
-		<li class"section"><a class"header" href="installconfig-install.html">Install</a>
+		<li class"section"><a class"header" href="installconfig-install.html">Installation</a>
 			<ul class="sub">
-				<li><a href="installconfig-install.html#installing-maas-from-ubuntu-server">Install from Ubuntu Server</a></li>
```

Comment:
This one is very vague to me. The word ISO should appear. Suggestion: "Install from Ubuntu Server ISO". I presume a change here would necessitate a change in the target page. The page's link looks fine.